client/blocks/product-selector/index.jsx

```diff
@@ -516,9 +544,9 @@ export class ProductSelector extends Component {
 					currencyCode={ currencyCode }
 					purchase={ purchase }
 					subtitle={ subtitle }
-					isCurrent={ isCurrent }
 				>
-					{ ! purchase && ! currentPlanIncludesProduct && (
+					{ hasProductPurchase && isCurrent && this.renderManageButton( purchase ) }
+					{ ! hasProductPurchase && ! isCurrent && (
```

Comment:
I wonder if we can make it more obvious that this condition is a negated version of the above one. A ternary?

Reply:
I don't think that a ternary would work here since those conditions are not really negated versions of each other. According to the De Morgan's law:

```
!a && !b = !(a || b)
!(a && b) = !a || !b
```

So in our case:

```
!a && !b ≠ !(a && b)
```